[2022-06-04 04:30:58,264] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 04:30:58,283] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 04:30:58,283] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 04:30:58,283] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 04:30:58,283] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 04:30:58,308] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 04:30:58,317] {standard_task_runner.py:52} INFO - Started process 34452 to run task
[2022-06-04 04:30:58,323] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpukolm0on', '--error-file', '/tmp/tmpfefvtl3d']
[2022-06-04 04:30:58,324] {standard_task_runner.py:80} INFO - Job 47: Subtask criar_table
[2022-06-04 04:30:58,409] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 04:30:58,515] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 04:30:58,532] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 04:30:58,535] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/uiliam/sandbox/lib/python3.9/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/uiliam/sandbox/lib/python3.9/site-packages/airflow/hooks/dbapi.py", line 186, in run
    with closing(self.get_conn()) as conn:
  File "/home/uiliam/sandbox/lib/python3.9/site-packages/airflow/providers/postgres/hooks/postgres.py", line 113, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/uiliam/sandbox/lib/python3.9/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "***" to address: Temporary failure in name resolution

[2022-06-04 04:30:58,550] {taskinstance.py:1396} INFO - Marking task as FAILED. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T073058, end_date=20220604T073058
[2022-06-04 04:30:58,568] {standard_task_runner.py:92} ERROR - Failed to execute job 47 for task criar_table (could not translate host name "***" to address: Temporary failure in name resolution
; 34452)
[2022-06-04 04:30:58,614] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-04 04:30:58,670] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 04:32:18,377] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 04:32:18,393] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 04:32:18,393] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 04:32:18,394] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 04:32:18,394] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 04:32:18,413] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 04:32:18,421] {standard_task_runner.py:52} INFO - Started process 35875 to run task
[2022-06-04 04:32:18,427] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpw_262vkn', '--error-file', '/tmp/tmp2m7712e3']
[2022-06-04 04:32:18,427] {standard_task_runner.py:80} INFO - Job 51: Subtask criar_table
[2022-06-04 04:32:18,507] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 04:32:18,608] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 04:32:18,622] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 04:32:18,628] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 04:32:18,629] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/uiliam/sandbox/lib/python3.9/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/uiliam/sandbox/lib/python3.9/site-packages/airflow/hooks/dbapi.py", line 193, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/uiliam/sandbox/lib/python3.9/site-packages/airflow/hooks/dbapi.py", line 217, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: table "census" does not exist

[2022-06-04 04:32:18,642] {taskinstance.py:1396} INFO - Marking task as FAILED. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T073218, end_date=20220604T073218
[2022-06-04 04:32:18,660] {standard_task_runner.py:92} ERROR - Failed to execute job 51 for task criar_table (table "census" does not exist
; 35875)
[2022-06-04 04:32:18,678] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-04 04:32:18,727] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 04:33:35,102] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 04:33:35,122] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 04:33:35,122] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 04:33:35,122] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 04:33:35,122] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 04:33:35,151] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 04:33:35,164] {standard_task_runner.py:52} INFO - Started process 36989 to run task
[2022-06-04 04:33:35,169] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmplder8fps', '--error-file', '/tmp/tmpog48osa3']
[2022-06-04 04:33:35,170] {standard_task_runner.py:80} INFO - Job 55: Subtask criar_table
[2022-06-04 04:33:35,252] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 04:33:35,362] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 04:33:35,376] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 04:33:35,382] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 04:33:35,387] {postgres.py:94} INFO - NOTICE:  table "census" does not exist, skipping

[2022-06-04 04:33:35,408] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T073335, end_date=20220604T073335
[2022-06-04 04:33:35,461] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 04:33:35,510] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 12:12:40,499] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:12:40,515] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:12:40,515] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:12:40,515] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 12:12:40,515] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:12:40,537] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 12:12:40,544] {standard_task_runner.py:52} INFO - Started process 17513 to run task
[2022-06-04 12:12:40,550] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp5h9_sl9a', '--error-file', '/tmp/tmp4tamn8gs']
[2022-06-04 12:12:40,550] {standard_task_runner.py:80} INFO - Job 61: Subtask criar_table
[2022-06-04 12:12:40,623] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 12:12:40,734] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 12:12:40,749] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 12:12:40,755] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 12:12:40,800] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T151240, end_date=20220604T151240
[2022-06-04 12:12:40,841] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 12:12:40,896] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 12:16:04,398] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:16:04,415] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:16:04,415] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:16:04,415] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 12:16:04,416] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:16:04,438] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 12:16:04,445] {standard_task_runner.py:52} INFO - Started process 20204 to run task
[2022-06-04 12:16:04,450] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpp2n61ymo', '--error-file', '/tmp/tmp6cx1wruj']
[2022-06-04 12:16:04,450] {standard_task_runner.py:80} INFO - Job 66: Subtask criar_table
[2022-06-04 12:16:04,525] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 12:16:04,627] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 12:16:04,643] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 12:16:04,648] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 12:16:04,674] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T151604, end_date=20220604T151604
[2022-06-04 12:16:04,701] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 12:16:04,755] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 12:22:52,462] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:22:52,479] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:22:52,479] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:22:52,479] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 12:22:52,480] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:22:52,500] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 12:22:52,507] {standard_task_runner.py:52} INFO - Started process 24307 to run task
[2022-06-04 12:22:52,511] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpqheledwt', '--error-file', '/tmp/tmpu9s6wcg2']
[2022-06-04 12:22:52,512] {standard_task_runner.py:80} INFO - Job 73: Subtask criar_table
[2022-06-04 12:22:52,586] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 12:22:52,691] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 12:22:52,704] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 12:22:52,710] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 12:22:52,738] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T152252, end_date=20220604T152252
[2022-06-04 12:22:52,803] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 12:22:52,863] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 12:26:22,902] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:26:22,917] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:26:22,918] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:26:22,918] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 12:26:22,918] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:26:22,937] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 12:26:22,948] {standard_task_runner.py:52} INFO - Started process 26097 to run task
[2022-06-04 12:26:22,953] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmph2hi6usa', '--error-file', '/tmp/tmpoxk_b12o']
[2022-06-04 12:26:22,954] {standard_task_runner.py:80} INFO - Job 79: Subtask criar_table
[2022-06-04 12:26:23,039] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 12:26:23,144] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 12:26:23,158] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 12:26:23,164] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 12:26:23,192] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T152622, end_date=20220604T152623
[2022-06-04 12:26:23,246] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 12:26:23,301] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 12:28:49,594] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:28:49,624] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:28:49,624] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:28:49,624] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 12:28:49,624] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:28:49,657] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 12:28:49,671] {standard_task_runner.py:52} INFO - Started process 27957 to run task
[2022-06-04 12:28:49,678] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpoe2l6u3h', '--error-file', '/tmp/tmpxh07a7pz']
[2022-06-04 12:28:49,679] {standard_task_runner.py:80} INFO - Job 85: Subtask criar_table
[2022-06-04 12:28:49,795] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 12:28:49,938] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 12:28:49,953] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 12:28:49,959] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 12:28:49,987] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T152849, end_date=20220604T152849
[2022-06-04 12:28:50,052] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 12:28:50,120] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 12:48:57,033] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:48:57,052] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 12:48:57,052] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:48:57,052] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 12:48:57,052] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 12:48:57,072] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 12:48:57,081] {standard_task_runner.py:52} INFO - Started process 36068 to run task
[2022-06-04 12:48:57,086] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmphgn3_bz3', '--error-file', '/tmp/tmp7qpirec1']
[2022-06-04 12:48:57,086] {standard_task_runner.py:80} INFO - Job 97: Subtask criar_table
[2022-06-04 12:48:57,165] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 12:48:57,277] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 12:48:57,295] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 12:48:57,300] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 12:48:57,329] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T154857, end_date=20220604T154857
[2022-06-04 12:48:57,377] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 12:48:57,435] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 13:17:36,373] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:17:36,391] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:17:36,391] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:17:36,391] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 13:17:36,391] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:17:36,414] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 13:17:36,424] {standard_task_runner.py:52} INFO - Started process 47876 to run task
[2022-06-04 13:17:36,430] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpmpx_3t24', '--error-file', '/tmp/tmpbnujj83i']
[2022-06-04 13:17:36,431] {standard_task_runner.py:80} INFO - Job 120: Subtask criar_table
[2022-06-04 13:17:36,516] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 13:17:36,632] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 13:17:36,650] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 13:17:36,657] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 13:17:36,687] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T161736, end_date=20220604T161736
[2022-06-04 13:17:36,721] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 13:17:36,786] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 13:33:09,557] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:33:09,574] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:33:09,575] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:33:09,575] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 13:33:09,575] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:33:09,597] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 13:33:09,604] {standard_task_runner.py:52} INFO - Started process 52254 to run task
[2022-06-04 13:33:09,610] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpaflh1k9p', '--error-file', '/tmp/tmpsr7fy2vz']
[2022-06-04 13:33:09,611] {standard_task_runner.py:80} INFO - Job 126: Subtask criar_table
[2022-06-04 13:33:09,695] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 13:33:09,801] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 13:33:09,817] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 13:33:09,823] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 13:33:09,853] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T163309, end_date=20220604T163309
[2022-06-04 13:33:09,901] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 13:33:09,960] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 13:47:46,336] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:47:46,356] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:47:46,357] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:47:46,357] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 13:47:46,357] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:47:46,382] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 13:47:46,394] {standard_task_runner.py:52} INFO - Started process 58380 to run task
[2022-06-04 13:47:46,400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpwjuv5pku', '--error-file', '/tmp/tmph3u6n3w_']
[2022-06-04 13:47:46,401] {standard_task_runner.py:80} INFO - Job 136: Subtask criar_table
[2022-06-04 13:47:46,486] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 13:47:46,634] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 13:47:46,670] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 13:47:46,678] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 13:47:46,714] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T164746, end_date=20220604T164746
[2022-06-04 13:47:46,771] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 13:47:46,866] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 13:58:11,374] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:58:11,393] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 13:58:11,394] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:58:11,394] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 13:58:11,394] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 13:58:11,421] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 13:58:11,430] {standard_task_runner.py:52} INFO - Started process 61451 to run task
[2022-06-04 13:58:11,436] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp727x70y0', '--error-file', '/tmp/tmpiw9riqn4']
[2022-06-04 13:58:11,436] {standard_task_runner.py:80} INFO - Job 142: Subtask criar_table
[2022-06-04 13:58:11,524] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 13:58:11,631] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 13:58:11,647] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 13:58:11,653] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 13:58:11,685] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T165811, end_date=20220604T165811
[2022-06-04 13:58:11,729] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 13:58:11,788] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 14:04:48,774] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:04:48,790] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:04:48,790] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:04:48,791] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 14:04:48,791] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:04:48,812] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 14:04:48,822] {standard_task_runner.py:52} INFO - Started process 64251 to run task
[2022-06-04 14:04:48,827] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpj042jqcc', '--error-file', '/tmp/tmpul_hnosi']
[2022-06-04 14:04:48,827] {standard_task_runner.py:80} INFO - Job 148: Subtask criar_table
[2022-06-04 14:04:48,903] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 14:04:49,005] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 14:04:49,021] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 14:04:49,027] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             INT NOT NULL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 14:04:49,055] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T170448, end_date=20220604T170449
[2022-06-04 14:04:49,119] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 14:04:49,180] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 14:19:42,109] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:19:42,126] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:19:42,127] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:19:42,127] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 14:19:42,127] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:19:42,149] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 14:19:42,159] {standard_task_runner.py:52} INFO - Started process 69347 to run task
[2022-06-04 14:19:42,163] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpsprn0xtk', '--error-file', '/tmp/tmpbp4_poei']
[2022-06-04 14:19:42,164] {standard_task_runner.py:80} INFO - Job 154: Subtask criar_table
[2022-06-04 14:19:42,243] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 14:19:42,360] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 14:19:42,377] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 14:19:42,383] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 14:19:42,416] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T171942, end_date=20220604T171942
[2022-06-04 14:19:42,455] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 14:19:42,515] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 14:30:54,236] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:30:54,253] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:30:54,253] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:30:54,253] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 14:30:54,253] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:30:54,275] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 14:30:54,282] {standard_task_runner.py:52} INFO - Started process 72985 to run task
[2022-06-04 14:30:54,286] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpv5vk8_ll', '--error-file', '/tmp/tmp7006bv2n']
[2022-06-04 14:30:54,287] {standard_task_runner.py:80} INFO - Job 160: Subtask criar_table
[2022-06-04 14:30:54,360] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 14:30:54,457] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 14:30:54,472] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 14:30:54,478] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 14:30:54,507] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T173054, end_date=20220604T173054
[2022-06-04 14:30:54,538] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 14:30:54,594] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 14:39:40,185] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:39:40,205] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 14:39:40,205] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:39:40,205] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 14:39:40,205] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 14:39:40,229] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 14:39:40,240] {standard_task_runner.py:52} INFO - Started process 77761 to run task
[2022-06-04 14:39:40,245] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp0fvg4sg_', '--error-file', '/tmp/tmpmaobzs9q']
[2022-06-04 14:39:40,246] {standard_task_runner.py:80} INFO - Job 174: Subtask criar_table
[2022-06-04 14:39:40,325] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 14:39:40,446] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 14:39:40,465] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 14:39:40,473] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 14:39:40,510] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T173940, end_date=20220604T173940
[2022-06-04 14:39:40,577] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 14:39:40,650] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 15:04:40,790] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:04:40,816] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:04:40,816] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:04:40,816] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 15:04:40,816] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:04:40,854] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 15:04:40,869] {standard_task_runner.py:52} INFO - Started process 83625 to run task
[2022-06-04 15:04:40,883] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '180', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp8af_z80y', '--error-file', '/tmp/tmpucfrwvjc']
[2022-06-04 15:04:40,884] {standard_task_runner.py:80} INFO - Job 180: Subtask criar_table
[2022-06-04 15:04:41,007] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 15:04:41,146] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 15:04:41,165] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 15:04:41,173] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 15:04:41,216] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T180440, end_date=20220604T180441
[2022-06-04 15:04:41,292] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 15:04:41,353] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 15:08:57,640] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:08:57,657] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:08:57,657] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:08:57,657] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 15:08:57,658] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:08:57,681] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 15:08:57,692] {standard_task_runner.py:52} INFO - Started process 85996 to run task
[2022-06-04 15:08:57,698] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpfsb1idz6', '--error-file', '/tmp/tmpsc3wcaxe']
[2022-06-04 15:08:57,699] {standard_task_runner.py:80} INFO - Job 186: Subtask criar_table
[2022-06-04 15:08:57,796] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 15:08:57,919] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 15:08:57,936] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 15:08:57,944] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 15:08:57,978] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T180857, end_date=20220604T180857
[2022-06-04 15:08:58,030] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 15:08:58,088] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 15:10:43,138] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:10:43,158] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:10:43,158] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:10:43,158] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 15:10:43,158] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:10:43,181] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 15:10:43,193] {standard_task_runner.py:52} INFO - Started process 87516 to run task
[2022-06-04 15:10:43,199] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp25tp27ma', '--error-file', '/tmp/tmpxajhgape']
[2022-06-04 15:10:43,199] {standard_task_runner.py:80} INFO - Job 192: Subtask criar_table
[2022-06-04 15:10:43,282] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 15:10:43,393] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 15:10:43,409] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 15:10:43,415] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 15:10:43,446] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T181043, end_date=20220604T181043
[2022-06-04 15:10:43,491] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 15:10:43,548] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 15:20:10,978] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:20:10,999] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:20:11,000] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:20:11,000] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 15:20:11,000] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:20:11,025] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 15:20:11,037] {standard_task_runner.py:52} INFO - Started process 90515 to run task
[2022-06-04 15:20:11,044] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpsi6yqr9s', '--error-file', '/tmp/tmpeegie348']
[2022-06-04 15:20:11,044] {standard_task_runner.py:80} INFO - Job 198: Subtask criar_table
[2022-06-04 15:20:11,144] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 15:20:11,265] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 15:20:11,282] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 15:20:11,289] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 15:20:11,325] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T182010, end_date=20220604T182011
[2022-06-04 15:20:11,377] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 15:20:11,440] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 15:27:00,202] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:27:00,220] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:27:00,220] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:27:00,220] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 15:27:00,220] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:27:00,242] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 15:27:00,254] {standard_task_runner.py:52} INFO - Started process 93082 to run task
[2022-06-04 15:27:00,259] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '205', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpr04ap8be', '--error-file', '/tmp/tmpwx1mihm6']
[2022-06-04 15:27:00,260] {standard_task_runner.py:80} INFO - Job 205: Subtask criar_table
[2022-06-04 15:27:00,340] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 15:27:00,448] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 15:27:00,464] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 15:27:00,471] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 15:27:00,502] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T182700, end_date=20220604T182700
[2022-06-04 15:27:00,551] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 15:27:00,606] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 15:42:44,888] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:42:44,906] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:42:44,907] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:42:44,907] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 15:42:44,907] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:42:44,930] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 15:42:44,941] {standard_task_runner.py:52} INFO - Started process 97527 to run task
[2022-06-04 15:42:44,947] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpx6mmcg9h', '--error-file', '/tmp/tmp7zriu5_n']
[2022-06-04 15:42:44,947] {standard_task_runner.py:80} INFO - Job 212: Subtask criar_table
[2022-06-04 15:42:45,026] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 15:42:45,136] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 15:42:45,152] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 15:42:45,159] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 15:42:45,191] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T184244, end_date=20220604T184245
[2022-06-04 15:42:45,238] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 15:42:45,296] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 15:49:50,162] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:49:50,179] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 15:49:50,179] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:49:50,179] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 15:49:50,179] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 15:49:50,205] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 15:49:50,216] {standard_task_runner.py:52} INFO - Started process 100328 to run task
[2022-06-04 15:49:50,222] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '218', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmped5syizg', '--error-file', '/tmp/tmpw8v7jni2']
[2022-06-04 15:49:50,223] {standard_task_runner.py:80} INFO - Job 218: Subtask criar_table
[2022-06-04 15:49:50,305] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 15:49:50,417] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 15:49:50,433] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 15:49:50,441] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 15:49:50,476] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T184950, end_date=20220604T184950
[2022-06-04 15:49:50,514] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 15:49:50,568] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 16:22:54,325] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 16:22:54,344] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 16:22:54,344] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 16:22:54,344] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 16:22:54,344] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 16:22:54,371] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 16:22:54,381] {standard_task_runner.py:52} INFO - Started process 107710 to run task
[2022-06-04 16:22:54,386] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpjah_czvz', '--error-file', '/tmp/tmpc2_sh3fh']
[2022-06-04 16:22:54,387] {standard_task_runner.py:80} INFO - Job 224: Subtask criar_table
[2022-06-04 16:22:54,468] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 16:22:54,575] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 16:22:54,591] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 16:22:54,597] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 16:22:54,630] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T192254, end_date=20220604T192254
[2022-06-04 16:22:54,678] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 16:22:54,730] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 18:30:05,718] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 18:30:05,730] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 18:30:05,730] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 18:30:05,730] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 18:30:05,730] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 18:30:05,746] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 18:30:05,751] {standard_task_runner.py:52} INFO - Started process 6157 to run task
[2022-06-04 18:30:05,755] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp7_3fvkl3', '--error-file', '/tmp/tmpislizj97']
[2022-06-04 18:30:05,755] {standard_task_runner.py:80} INFO - Job 267: Subtask criar_table
[2022-06-04 18:30:05,818] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 18:30:05,894] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 18:30:05,909] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 18:30:05,914] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 18:30:05,955] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T213005, end_date=20220604T213005
[2022-06-04 18:30:06,006] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 18:30:06,053] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 19:05:54,338] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 19:05:54,356] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 19:05:54,356] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 19:05:54,356] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 19:05:54,356] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 19:05:54,380] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 19:05:54,389] {standard_task_runner.py:52} INFO - Started process 8567 to run task
[2022-06-04 19:05:54,394] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '273', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpzzj9gyix', '--error-file', '/tmp/tmpujxwp82c']
[2022-06-04 19:05:54,394] {standard_task_runner.py:80} INFO - Job 273: Subtask criar_table
[2022-06-04 19:05:54,475] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 19:05:54,595] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 19:05:54,611] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 19:05:54,618] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 19:05:54,662] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T220554, end_date=20220604T220554
[2022-06-04 19:05:54,726] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 19:05:54,787] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-04 19:16:57,839] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 19:16:57,864] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 19:16:57,865] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 19:16:57,865] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 19:16:57,865] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 19:16:57,895] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 19:16:57,905] {standard_task_runner.py:52} INFO - Started process 12339 to run task
[2022-06-04 19:16:57,912] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpv80v49jr', '--error-file', '/tmp/tmp2whav_39']
[2022-06-04 19:16:57,913] {standard_task_runner.py:80} INFO - Job 278: Subtask criar_table
[2022-06-04 19:16:58,019] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 19:16:58,152] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 19:16:58,173] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 19:16:58,179] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 19:16:58,219] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T221657, end_date=20220604T221658
[2022-06-04 19:16:58,284] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 19:16:58,368] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 19:18:47,300] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 19:18:47,317] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 19:18:47,317] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 19:18:47,317] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 19:18:47,318] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 19:18:47,338] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 19:18:47,348] {standard_task_runner.py:52} INFO - Started process 13953 to run task
[2022-06-04 19:18:47,354] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '284', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpmidkvw9d', '--error-file', '/tmp/tmp0z_7auy3']
[2022-06-04 19:18:47,355] {standard_task_runner.py:80} INFO - Job 284: Subtask criar_table
[2022-06-04 19:18:47,435] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 19:18:47,543] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 19:18:47,559] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 19:18:47,565] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 19:18:47,597] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220604T221847, end_date=20220604T221847
[2022-06-04 19:18:47,646] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 19:18:47,705] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-04 21:17:25,239] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 21:17:25,332] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-06-04 21:17:25,333] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 21:17:25,334] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 21:17:25,334] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 21:17:25,432] {taskinstance.py:1378} INFO - Executing <Task(PostgresOperator): criar_table> on 2022-06-03 00:00:00+00:00
[2022-06-04 21:17:25,468] {standard_task_runner.py:52} INFO - Started process 35691 to run task
[2022-06-04 21:17:25,484] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'criar_table', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp6tu4ufyi', '--error-file', '/tmp/tmp4of6gbfk']
[2022-06-04 21:17:25,485] {standard_task_runner.py:80} INFO - Job 319: Subtask criar_table
[2022-06-04 21:17:25,712] {task_command.py:370} INFO - Running <TaskInstance: census.criar_table scheduled__2022-06-03T00:00:00+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-04 21:17:25,965] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=criar_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-06-04 21:17:26,007] {base.py:68} INFO - Using connection ID '***_default' for task execution.
[2022-06-04 21:17:26,030] {dbapi.py:213} INFO - Running statement: 
            DROP TABLE IF EXISTS census;
            CREATE TABLE census (
                id_census             SERIAL PRIMARY KEY,
                age                   INT,
                workclass             VARCHAR(20),
                fnlwgt                FLOAT,
                education             VARCHAR(20),
                education_num         INT,
                marital_status        VARCHAR(30),
                occupation            VARCHAR(20),
                relationship          VARCHAR(30),
                race                  VARCHAR(30),
                sex                   VARCHAR(10),
                capital_gain          FLOAT,
                capital_loss          FLOAT,
                hours_per_week        INT,
                native_country        VARCHAR(30),
                class                 VARCHAR(10)
                )
        , parameters: None
[2022-06-04 21:17:26,312] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=criar_table, execution_date=20220603T000000, start_date=20220605T001725, end_date=20220605T001726
[2022-06-04 21:17:26,420] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 21:17:26,579] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
